From e65cf7406f259540d831315a6fed3fb8291aced7 Mon Sep 17 00:00:00 2001
From: Adrian Huang <ahuang@redhat.com>
Date: Wed, 29 Oct 2014 16:29:30 -0400
Subject: [redhat] RHMAINTAINERS: Add Adrian Huang to Lenovo entry

Message-id: <1414600170-3763-1-git-send-email-ahuang@redhat.com>
Patchwork-id: 98518
O-Subject: [RHEL7.1 PATCH] RHMAINTAINERS: Add myself to a Lenovo entry
Bugzilla: 1155108
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1155108

Add myself to a Lenovo entry so that Lenovo can get the notification.

Signed-off-by: Adrian Huang <ahuang@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index bb71627..3d11e24 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -362,6 +362,11 @@ M: Neil Horman <nhorman@redhat.com>
 S: Maintained
 F: drivers/i2c/busses/i2c-ismt.c
 
+Lenovo Hardware Support
+M: Adrian Huang <ahuang@redhat.com>
+S: Maintained
+F: drivers/gpu/drm/ast/
+
 LSI 3ware SAS/SATA-RAID Linux Drive (3w-sas)
 M: Tomas Henzl <thenzl@redhat.com>
 S: Maintained
-- 
1.7.1